labelbox/schema/asset_attachment.py

```diff
@@ -19,6 +19,8 @@ class AttachmentType(Enum):
         TEXT = "TEXT"
         IMAGE_OVERLAY = "IMAGE_OVERLAY"
         HTML = "HTML"
+        RAW_TEXT = "RAW_TEXT"
+        TEXT_FILE = "TEXT_FILE"
```

Comment:
Two thoughts:
1. What about calling this `TEXT_URL` - as that better reflects what users will be providing?
2. Unfortunately, having to qualify with `_FILE` now makes this one inconsistent with the other values, which all expect a URL but don't have a suffix `_URL`. I don't suppose there is a good workaround, besides eventually making all of the others suffixed with `_URL` (or similar). <sigh> Not requesting any change just wondering if you have any thoughts on this.